packages/marketplace/src/components/ui/submit-app-wizard/steps/step-chose-auth-type.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import styles from '@/styles/blocks/submit-app-wizard.scss?mod'
+import { ModalBody, Button, ModalFooter, H4 } from '@reapit/elements'
+import { WizardStepComponent, SetWizardStep } from '../types'
+import { formFields } from '../form-fields'
+import { useFormikContext } from 'formik'
+
+const { authFlowField } = formFields
+
+export const onClientSide = (setWizardStep: SetWizardStep, setFieldValue) => () => {
+  setFieldValue(authFlowField.name, 'authorisationCode')
+  setWizardStep('INPUT_AUTHENTICATION_URIS')
+}
+
+export const onServerSide = (setWizardStep: SetWizardStep, setFieldValue) => () => {
+  setFieldValue(authFlowField.name, 'clientCredentials')
+  setWizardStep('GRANT_PERMISSION')
+}
+
+export const onPrev = (setWizardStep: SetWizardStep) => () => {
+  setWizardStep('CREATE_NEW_APP')
```

Comment:
I think you should assign these step string as `const` instead
E.g.
```
const SUBMIT_STEPS = {
  GRANT_PERMISSION: 'GRANT_PERMISSION',
  INPUT_AUTHENTICATION_URIS: 'INPUT_AUTHENTICATION_URIS'
  //others
}
```
so we can have IntelliSense and avoid typo in the future